docs/tutorial/security.md

```diff
@@ -782,7 +782,10 @@ provided by the main process modules may be dangerous in the hands of
 malicious code running in a compromised renderer process. By limiting
 the set of accessible modules to the minimum that your app needs and
 filtering out the others, you reduce the toolset that malicious code
-can use to attack the system.
+can use to attack the system. Note that you'll still be vulnerable to
+"prototype pollution" attacks on the remote module internal mechanism,
```

Comment:
Hm, to me this implies that there are known prototype pollution bugs in the remote module mechanism, which isn't the case. Perhaps:

> Note that the safest option is to [fully disable the remote module](#15-disable-the-remote-module). If you choose to filter access rather than completely disable the module, you must be very careful to ensure that no escalation of privilege is possible through the modules you allow past the filter.

Reply:
Agreed, using now your suggestion.